### PR TITLE
New package: tremc-0.9.1

### PR DESCRIPTION
**Pull request by natemonster321**

### CODE REVIEW

File: srcpkgs/tremc/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+build_style=gnu-makefile
+hostmakedepends="python3 git"
+makedepends="python3"
+depends="python3"
+short_desc="A console client for the BitTorrent client Transmission"
+maintainer="Nathan D <nathan.dhanas@gmail.com>"
+license="GPL-3.0"
+homepage="https://github.com/tremc/tremc"
+
+do_fetch() {
```
Comment (Johnnynator) [inline srcpkgs/tremc/template]: Nope. You should use the release tarball `https://github.com/tremc/tremc/archive/0.9.1.tar.gz`

File: srcpkgs/tremc/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'tremc'
+pkgname=tremc
+version=0.9.1
+revision=1
+wrksrc="${pkgname}-${version}"
```
Comment (Johnnynator) [inline srcpkgs/tremc/template]: This is the default. You don't have to add this line.

File: srcpkgs/tremc/template [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+build_style=gnu-makefile
+hostmakedepends="python3 git"
+makedepends="python3"
+depends="python3"
+short_desc="A console client for the BitTorrent client Transmission"
+maintainer="Nathan D <nathan.dhanas@gmail.com>"
+license="GPL-3.0"
```
Comment (Johnnynator) [inline srcpkgs/tremc/template]: Pls use [SPDX-Identifier](https://spdx.org/licenses/) (version 3 or newer)

File: srcpkgs/tremc/template [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'tremc'
+pkgname=tremc
+version=0.9.1
+revision=1
```
Comment (Johnnynator) [inline srcpkgs/tremc/template]: You can add `noarch=yes` after this line.

File: srcpkgs/tremc/template [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+wrksrc="${pkgname}-${version}"
+build_style=gnu-makefile
+hostmakedepends="python3 git"
+makedepends="python3"
```
Comment (Johnnynator) [inline srcpkgs/tremc/template]: `python3` in `makedepends` and `hostmakedepends` is not needed.

**Comment by maxice8** [comment]
put everything into 1 commit

**Pull request merged**